src/analyses/ai.h

```diff
@@ -18,6 +18,7 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <util/json.h>
 #include <util/xml.h>
 #include <util/expr.h>
+#include <util/make_unique.h>
```

Comment:
`util/make_unique.h` itself includes `memory`. My preference would be to leave it as-is, and then if we change to C++14 we can find-and-replace `util/make_unique -> memory` and `util_make_unique -> std::make_unique`.